### ISSUE TITLE
Windows 2.x/386 Unsupported DOS version

### ISSUE THREAD

**Issue by rderooy** [joncampbell123/dosbox-x#3047]
### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

### Have you checked that no other similar bug report(s) already exists?

- [X] I have searched and didn't find any similar issues.

### What operating system(s) this bug have occurred on?

Linux F35

### What version(s) of DOSBox-X have this bug?

0.83.20git

### Describe the bug

Trying to run a Windows 2.x/386 edition in DOSBox-X (without booting DOS) gives the error:

```
C:\WIN386>WIN386.EXE
Error: Unsupported DOS version

C:\WIN386>
```
This happens regardless of the DOS version set. It should work with DOS  >= 3.10 or <5.0
I tried various memory and a20 options without success.

Tested with Windows 2.01/386 and 2.11/386, and both have the same issue.

The Windows 2.03 version does work, while the Windows 2.11/286 version has an issue with hanging on the title screen.

### Steps to reproduce the behaviour

see above

### Used configuration

```ini
[sdl]
autolock=true

[dosbox]
title=Windows/386 2.01
memsize=2

[cpu]
cputype=386

[dos]
ver=4.0
ems=false
vcpi=false
xms=false
umb=false
hma=false

[dosbox]
a20=off

[serial]
serial1=serialmouse

[parallel]
parallel1=printer

[printer]
multipage=true
timeout=5000

[render]
scaler=none

[config]
dos=low

[autoexec]
mount c .
c:
IF EXIST C:\WIN386\WIN386.EXE GOTO WIN386
IMGMOUNT A Disk3.img
CD INSTALL
SETUP

:WIN386
SET PATH=%PATH%;C:\WIN386
C:\WIN386\WIN386.EXE
GOTO END

:END
```

**Comment by joncampbell123** [comment]
I can confirm this happens but I have no idea why, yet.

**Comment by Siddhant-Mashalkar** [comment]
i installed Windows 2.x/386 on DOS 3.31 Hard Disk Image (32Mb) , it starts but mouse doesn't work

**Comment by rderooy** [comment]
Yes, installing Windows 2.x/386 in a HD image with DOS works, this issue is about running it directly without.

And the mouse is a known issue, mentioned on the wiki. Either use the serialmouse or the "Microsoft Mouse connected to PS/2 Mouse Port", depending on the exact Windows version. For some reason the "IBM Personal Systems/2 Mouse" does not work.

**Comment by rderooy** [comment]
Just tested the 2025.01.01 release, and this issue is still current.
